# HG changeset patch
# User Alexander Dreyer <adreyer@gmx.de>
# Date 1332106337 -3600
# Node ID cab5bead97560e1971cef33f2b06aed71d97d99f
# Parent  8844c3db55b1e3e49c56ee563b14136582955d24
FIX: prerequisites check for testing plot.py

diff -r 8844c3db55b1e3e49c56ee563b14136582955d24 -r cab5bead97560e1971cef33f2b06aed71d97d99f pyroot/polybori/plot.py
--- a/pyroot/polybori/plot.py	Sun Mar 18 22:04:48 2012 +0100
+++ b/pyroot/polybori/plot.py	Sun Mar 18 22:32:17 2012 +0100
@@ -24,8 +24,11 @@
             return False
 
     try:
-        process = Popen(["dot", "-V"])
-        process.close()               
+        from subprocess import Popen, PIPE
+        from os import devnull
+        out = open(devnull)
+        process = Popen(["dot", "-V"], stderr=out, stdout=out)
+        out.close()
     except:                           
         return False   
 
